app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        return self.skills.append(skill)
```

Comment:
The `learn_skill` method should not return the result of `self.skills.append(skill)`. Since `append` modifies the list in place and returns `None`, the method should not return anything. Simply use `self.skills.append(skill)` without `return`.